core/src/main/java/tc/oc/pgm/regions/RegionMatchModule.java

```diff
@@ -404,19 +405,13 @@ private ParticipantState getActor(BlockTransformEvent event) {
     return ParticipantBlockTransformEvent.getPlayerState(event);
   }
 
-  private static BlockState getHangingBlockState(Hanging hanging) {
-    Block block = hanging.getLocation().getBlock();
-    Material type = getHangingType(hanging);
-    return type == null ? null : BlockStates.cloneWithMaterial(block, type);
-  }
-
-  private static Material getHangingType(Hanging hanging) {
+  private static MaterialData getHangingType(Hanging hanging) {
     if (hanging instanceof Painting) {
-      return Material.PAINTING;
+      return MaterialData.of(Material.PAINTING);
     } else if (hanging instanceof ItemFrame) {
-      return Material.ITEM_FRAME;
+      return MaterialData.of(Material.ITEM_FRAME);
```

Comment:
Probably need to handle [glow item frames](https://minecraft.wiki/w/Glow_Item_Frame)